src/add.spec.js

```diff
@@ -0,0 +1,26 @@
+import add from "./add"
+
+// The add function should be tested to ensure that the addition of the provided integers & 
+// floating-point numbers provides the mathematically correct result. 
+
+describe('Test suite for "add"-function', () => {
```

Comment:
I would usually just write "add". To create a sentence with the it() after. e.g. "add should return 4..."
```suggestion
describe('add', () => {
```